### DISCUSSION TITLE
Text-to-speech from visual mode

### DISCUSSION THREAD

**Discussion by Theogoreux** [tridactyl/tridactyl#5002] [Q&A] [answered]
Hello, I noticed it is possible to read links with TTS with ;r. Is it possible to read any text highlighted in visual mode ? Thank you

**Comment by bovine3dom** [comment, marked as answer]
Untested but e.g.

`:bind --mode=visual r composite js document.getSelection().toString() | ttsread -t`

should work

**Comment by Theogoreux** [comment]
It works perfectly thanks you very much
